README.md

```diff
@@ -1,48 +1,248 @@
 # MESA GUI
 
-Micro Evaluation Security Assessments (MESAs) are designed to provide organizations with insights into their internal security posture. The MESA-GUI serves as a user-friendly front-end for the MESA-Toolkit command-line interface (https://github.com/cisagov/mesa-toolkit/). This project relies on the successful installation of the MESA-Toolkit and is not intended to function as a standalone application.
+Micro Evaluation Security Assessments (MESAs) are designed to provide organizations with insights into their internal security posture. The MESA-GUI serves as a user-friendly front-end for the [MESA-Toolkit](https://github.com/cisagov/mesa-toolkit/) command-line interface. This project relies on the successful installation of the MESA-Toolkit and is not intended to function as a standalone application.
 
 It is important to note that MESAs are not designed to furnish a comprehensive understanding of the entire internal environment. Instead, their purpose is to equip organizations with the essential information needed to establish a foundational security posture. This emphasis revolves around safeguarding against commonly exploited misconfigurations and vulnerabilities. MESAs lay the groundwork for organizations to initiate the process of fortifying their security stance, ensuring a proactive defense against prevalent risks.
 
-# Getting started
+## Installing the MESA-GUI
+>[!IMPORTANT]
+>It is recommended to use the [MESA-Toolkit](https://github.com/cisagov/mesa-toolkit/) installation bash script prior to installing the MESA-GUI to maintain a standardized naming convention for resource locations.
 
-Once the MESA-Toolkit has been successfully installed on a `Debian 12.7.0` instance, the following commands can be used to manually setup the MESA-GUI:
+Once the [MESA-Toolkit](https://github.com/cisagov/mesa-toolkit/) has been successfully installed on a `Debian 12.7.0` instance, the following steps can be used to manually setup the MESA-GUI.
+
+1. **Clone the MESA-GUI repository**
+   
+   Clone this repository into the /opt/ directory to download the MESA-GUI source code locally:
 
 ```bash
-git clone https://github.com/cisagov/mesa-gui
+cd /opt/
+sudo git clone https://github.com/cisagov/mesa-gui
 cd mesa-gui
+```
+
+2. **Source the MESA virtual environment**
 
-# create virtual environment
-python3 -m venv venv
-. ./venv/bin/activate
+   Source the MESA virtual environment. This activates the environment, isolating the dependencies needed for the MESA-GUI to run:
+
+```bash
+source /opt/MESA-venv/bin/activate
```

Comment:
This is created by the mesa-toolkit installer right? Might be worth mentioning as a bash comment, but if this is walking through the installation as if mesa-toolkit was not used, should we also include creating the venv?

Reply:
Good call, if it's going to be a manual install then the `venv` will need to be created.